docs/submitting-clinical-data.md

```diff
@@ -14,17 +14,15 @@ Clinical data consists of all supporting data about your program's donors and th
 1. Upon uploading, if there are any errors in your file they will be displayed within the Clinical Submission workspace. The error report will also be available for download. You must fix all of the errors that are listed within your file(s) and then reupload them.
 1. Valid files will be available for review within the file preview tables. You can review the file data for each data type using by using the tabs on the left of the preview area.
 
-## Completing Your Data Submission
-
-### Validating Clinical Data Files
-Uploaded data must be checked for accuracy against both the data you have currently uploaded and the data that has previously been submitted for your program.
+## Validating Clinical Data Files
+Uploaded data must be checked for accuracy against the data in the current submission, as well as any data that has previously been submitted for your program.
 
 1. Once you are ready, click the **Validate Submission** button to check your entire submission workspace for data consistency.
 1. Upon validating, if there are any errors in your submission they will be displayed within the submission workspace. The error report will also be available for download. You must fix all of the errors that are listed within your file(s) and then reupload them.
 1.  If there are no errors in your submission, then your submission will be marked as _Valid_.
 
 
-### Signing Off on Your Submission
-1. Once you have reviewed the files in the file preview area, click the **Sign Off Submission** button to submit your clinical data.
+## Signing Off on Your Submission
+Once you have reviewed the files in the file preview area, click the **Sign Off Submission** button to submit your clinical data. If you have made any updated to data that has already been submitted, you clinical workspace will be placed in a Pending Approval state.  A DCC member will review your updates and approve them for submission.  If there are any issues, your submission will be reopened and the DCC will notify you of any changes needed.
```

Comment:
changed "any updated" to "any updates"

do we want to say "you clinical workspace will be locked in a Pending Approval state"

change last bit to "be reopened and the DCC will notify you, via email, of the required changes."